.devcontainer/devcontainer.json

```diff
@@ -4,13 +4,11 @@
 	"image": "j00sko/programiranje1:latest",
 	
 	// Build the image locally 
-	/*
```

Comment:
Tega raje ne bi dal za privzeto, ker traja precej časa. Ali je kakšen razlog, da vnaprej zgrajen image ni dober?

Reply:
Aha, razumem. Motiv za spreminjanje privzete nastavitve je bil, da mi brez odkomentiranja OCaml language extension za VSCode ni delal.

Reply:
To bi lokalno vsekakor odkomentiral, da lahko preizkušate spremembe, ne bi pa tega pocomittal. Bi bilo pa komentar smiselno razširiti s temi navodili.